tests/detect-bypass/test.yaml

```diff
@@ -0,0 +1,22 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 1
```

Comment:
Answering your comment:
indeed, my understanding from the test is also that sid 1 should not alert.
And that we want sid 3 to alert.